isic/core/tests/test_doi.py

```diff
@@ -8,9 +8,11 @@
 
 @pytest.fixture
 def mock_datacite_create_doi(mocker):
-    yield mocker.patch(
```

Comment:
We weren't actually making any assertions that this was being called. Should we? If so I can make these two separate fixtures and add some assertions.

Reply:
> If so I can make these two separate fixtures and add some assertions.

Sounds good to me.